### PR TITLE
Add jenkins release pipeline

### PR DESCRIPTION
**Pull request by wanglinsong**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            yaml AGENT_YAML
+        }
+    }
+
+    environment {
+        GPG_SECRET     = credentials('presto-release-gpg-secret')
+        GPG_TRUST      = credentials("presto-release-gpg-trust")
+        GPG_PASSPHRASE = credentials("presto-release-gpg-passphrase")
+
+        GITHUB_OSS_TOKEN_ID = 'github-personal-token-wanglinsong'
```
Comment (tdcmeehan) [inline Jenkinsfile:34]: Use a bot account please

**Review by tdcmeehan — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (wanglinsong) [reply Jenkinsfile:34]: I will rename this, but the underline credential is still from my account, unless you share another github API token with me.

Comment (wanglinsong) [reply Jenkinsfile]: updated.

**Comment by tdcmeehan** [comment]
Please squash commits

**Pull request merged**